test/datahike/test/utils.cljc

```diff
@@ -26,3 +26,23 @@
 (defn all-true? [c] (every? true? c))
 
 (defn all-eq? [c1 c2] (all-true? (map = c1 c2)))
+
+(defn setup-default-db [config test-data]
+  (let [_ (d/delete-database config)
+        _ (d/create-database config)
+        conn (d/connect config)]
+    (d/transact conn test-data)
+    conn))
+
+(defn teardown-db [conn]
+  (d/release conn)
+  (d/delete-database (:config @conn)))
+
+(defn with-db
```

Comment:
I like these utilities, at some point we would need again some refactoring of our test setup.